### ISSUE TITLE
hotp-verification hash upstream points to later available commit while nitrokey fork points to 1.4?

### ISSUE THREAD

**Issue by tlaurion** [Nitrokey/heads#49]
Also, hotp-verification hash upstream points to later available commit while nitrokey fork points to 1.4?

Upstream:
https://github.com/linuxboot/heads/blob/ee1978ffc0afa4db1b0a46517aff946912cdca14/modules/hotp-verification#L5-L10
 
Nitrokey:
https://github.com/Nitrokey/heads/blob/da154df82c61f5b3ff2baad540f55500da2373d1/modules/hotp-verification#L5-L10

Was the latest hotp-verification code not aimed to be used upstream @daringer ?

_Originally posted by @tlaurion in https://github.com/Nitrokey/heads/issues/48#issuecomment-2050330189_

**Comment by daringer** [comment]
the heads upstream version is v1.5 - this is all correct, and functionally identical - just not perfectly clean executed....

this (34b47aa75c07522d416c915911eed820556f1da8) is functionally identical, just without the updated version string, which is done here: e9d8082af43a2cdc20c08f78cb2d59f30d4f4076 and then released as v1.5 using the commit you see upstream: 70c04f51387eee8f777e943ba83b6405764a3cd2

**Comment by tlaurion** [comment]
@daringer downstram and upstream should be limited to changes waiting to be part of an upstream commit outside of branding AFAIK?

**Comment by daringer** [comment]
yep, you are totally right, we are working on the next release already - there it will be fixed
